### ISSUE TITLE
Spaced indents?

### ISSUE THREAD

**Issue by dsherret** [dsherret/dprint#114]
From #105. Need to figure out how to deal with this:

```ts
return currentWidthToHeight > targetWidthToHeight
    ? // window width is too wide relative to desired game width
      [currentHeight * targetWidthToHeight, currentHeight]
    : // window height is too high relative to desired game height
      [currentWidth, currentWidth / targetWidthToHeight];
```

What should happen when the indent width is `4` instead of `2`? Might need to add something into the printer to handle this so it always does 2 indents, but need to research this more.

**Comment by hayd** [comment]
> What should happen when the indent width is 4 instead of 2? 

It should indent 4 in that case. (I don't see any argument for it always doing 2.)

**Comment by dsherret** [comment]
@hayd I think so as well and that would be less complex. Prettier does 2 and when tabs are selected it also does 2 spaces... that is, until the inner part is indented and then it just does tabs (which seems like a bug if it was going for having 2 spaces there).

I think I'm just going to indent for now and consider changing this in the future. It is strange in the editor when lines don't line up with the expected indent location:

![image](https://user-images.githubusercontent.com/1609021/75099811-0ffd4a00-5594-11ea-8042-003af00b447a.png)

**Comment by hayd** [comment]
Bizarre! I don't think you should blindly match prettier here. Most people use 2 spaces anyways. 👍

**Comment by dsherret** [comment]
Resolved the formatting issue from #105 that's outlined in this issue. Closing this as "won't do" though.
